### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by radishevskyi**
# 📝 Description

- added section analytics

# 🔄 Type of change

- [ ] Cleaning up
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality to something)
- [x] Configuration change
- [x] Translation change

# 🧪 How has this been tested / or will it be tested?

- [ ] Manually surfing the app

# ✅ Checklist:

- [x] My changes generate no new warnings in console or terminal
- [x] I have performed a self-review of my code
- [x] I have updated versions of the package.json and package-lock.json

# 🖼️ Result screenshot/video (optional): 
image

# Gif to summarize my emotions (optional):

### CODE REVIEW

File: src/modules/home/analytics/analytics.styles.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  AnalyticsBlock: styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: var(--spacing-x2);
+  `,
+
+  Title: styled.span`
+    font-size: var(--font-size-title);
+    background-image: var(--text-background-gradient);
```
Comment (p-osypov) [inline src/modules/home/analytics/analytics.styles.ts:35]: If this patter is duplicated many times then it should be separate component.
Create a component Title under components folder. Update other titles that having these styles

File: src/modules/home/analytics/analytics.styles.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    gap: calc(var(--spacing) * 6);
+    flex-wrap: wrap;
+  `,
+
+  DestroyedEquipBlock: styled.div`
+    display: flex;
+    align-items: center;
+    flex: 1;
+    position: relative;
+    min-width: 562px;
```
Comment (p-osypov) [inline src/modules/home/analytics/analytics.styles.ts:52]: This can't be flexible. Update media styles

**Review by p-osypov — CHANGES_REQUESTED** [review]
Finish adaptive design. 
Make the cards same height in a line. 
Reduce font size and gaps for tablet/mobile
Screenshot 2024-12-10 at 19 12 09
Screenshot 2024-12-10 at 19 07 32
Screenshot 2024-12-10 at 19 03 33

File: src/modules/home/analytics/analytics.styles.ts (hunk 1 above)
Comment (radishevskyi) [reply src/modules/home/analytics/analytics.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/8/commits/e7044702eb2b08bd467879e1a2b4171b43251598)

File: src/modules/home/analytics/analytics.styles.ts (hunk 2 above)
Comment (radishevskyi) [reply src/modules/home/analytics/analytics.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/8/commits/e7044702eb2b08bd467879e1a2b4171b43251598)

**Review by p-osypov — APPROVED** [review]

**Pull request merged**